### PR TITLE
Kpi two

### PR DESCRIPTION
**Pull request by drudge77**
Added Exception Handling
Registered remaining models to Admin

### CODE REVIEW

File: dashboard/admin.py [hunk 1]
```diff
@@ -24,6 +24,13 @@
 from dashboard.models import OrganisationZipcode
 from dashboard.models import OrganisationZipcodeAdmin
 
+<<<<<<< HEAD
```
Comment (PraveenKumarPalai) [inline dashboard/admin.py]: Please resolve conflicts and push

File: dashboard/controllers/organisation.py [hunk 2]
```diff
@@ -77,32 +82,37 @@ def get_queryset(self, db, level, radius):
...
-        qs = self.get_queryset(db, level, radius)
-        data = serializer_class(qs, many=True).data
-        msg = f"Data for Level {level} found"
-        context = response_format(success=True, message=msg, data=data)
-        return Response(context, status.HTTP_200_OK)
-
-
+        level = int(request.GET.get('level', 1))
+        try:
+            if level == 1:
```
Comment (PraveenKumarPalai) [inline dashboard/controllers/organisation.py]: Proceed with a modular approach. Use switch case here..

``` switch```

create  a separate function using:

```  _functionName ```

File: dashboard/controllers/kpi_two.py [hunk 3]
```diff
@@ -22,36 +22,49 @@ class RequestInfo(APIView):
...
-        context = response_format(success=True, message=msg, data={})
-        return Response(context, status.HTTP_200_OK)
+        try:
+            data_id = request.GET['id']
+            obj = data[int(data_id)]
+            domain = getattr(settings, 'MAILGUN_DOMAIN')
+            api_key = getattr(settings, 'MAILGUN_API_KEY')
+            email = 'satyam.saxena@cyberboxer.com'  # Debug Only
+            res = requests.post(
+                f"https://api.mailgun.net/v3/{domain}/messages",
```
Comment (PraveenKumarPalai) [inline dashboard/controllers/kpi_two.py]: What is f ??

File: dashboard/controllers/organisation.py (hunk 2 above)
Comment (drudge77) [reply dashboard/controllers/organisation.py]: > Proceed with a modular approach. Use switch case here..
> 
> ` switch`
> 
> create a separate function using:
> 
> ` _functionName`

Should I create 4 different functions and map them using a dictionary?

File: dashboard/controllers/kpi_two.py (hunk 3 above)
Comment (drudge77) [reply dashboard/controllers/kpi_two.py]: > What is f ??

f represents the string being a **_f string_**. In these strings the values inside braces {} are automatically evaluated. So its a short version of writing 
`"https://api.mailgun.net/v3/" + domain + "messages"`  
or  
`"https://api.mailgun.net/v3/{0}/messages".format(domain)`

Comment (PraveenKumarPalai) [reply dashboard/controllers/kpi_two.py]: Okay got that

File: dashboard/controllers/organisation.py (hunk 2 above)
Comment (PraveenKumarPalai) [reply dashboard/controllers/organisation.py]: Create a separate file for the if else condition and use ````switch```` instead of ````if ````

**Pull request merged**